discovery_engine/lib/src/domain/engine/engine.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:xayn_discovery_engine/src/domain/models/active_data.dart'
+    show ActiveDocumentData;
+import 'package:xayn_discovery_engine/src/domain/models/document.dart'
+    show Document;
+
+/// Proxy to Discovery Engine core.
+abstract class Engine {
+  /// Retrieves at most [maxDocuments] feed documents.
+  Map<Document, ActiveDocumentData> getFeedDocuments(int maxDocuments);
```

Comment:
How does that relate to the different stacks, do we need to pass in the choice of stack?

Reply:
I believe the stacks are a matter dealt with on the rust side rather than here, but I'll defer to @acrrd  on that